fenix/app/src/androidTest/java/org/mozilla/fenix/ui/OnboardingTest.kt

```diff
@@ -27,6 +28,7 @@ class OnboardingTest {
     }
 
     // TestRail link: https://testrail.stage.mozaws.net/index.php?/cases/view/2122334
+    @Ignore("Disabled: https://bugzilla.mozilla.org/show_bug.cgi?id=1882038")
```

Comment:
❓ Could we update the tests to match the new spec?

Reply:
I actually tried together with @AndiAJ to figure out a way to update the tests and currently the best method we have is to build a `testTag` for both primary and secondary buttons of the `OnboardingPage` like this:
```
testTag = pageState.title + "onboarding_card.positive_button"
 and 
testTag = pageState.title + "onboarding_card.negative_button"
```
and add in `HomeScreenRobot` dedicated methods for each of these buttons present on each onboarding page:

```
    fun clickDefaultCardNotNowOnboardingButton(composeTestRule: ComposeTestRule) {
        Log.i(TAG, "clickNotNowOnboardingButton: Trying to click \"Not now\" onboarding button")
        composeTestRule.onNodeWithTag(
            getStringResource(R.string.juno_onboarding_default_browser_title_nimbus_2) + "onboarding_card.negative_button",
        ).performClick()
        Log.i(TAG, "clickNotNowOnboardingButton: Clicked \"Not now\" onboarding button")
    }

    fun clickAddSearchWidgetNotNowOnboardingButton(composeTestRule: ComposeTestRule) {
        Log.i(TAG, "clickNotNowOnboardingButton: Trying to click \"Not now\" onboarding button")
        composeTestRule.onNodeWithTag(
            getStringResource(R.string.juno_onboarding_add_search_widget_title) + "onboarding_card.negative_button",
        ).performClick()
        Log.i(TAG, "clickNotNowOnboardingButton: Clicked \"Not now\" onboarding button")
    }
``` 

Added the changes in the PR.

I also thought of adding actual test tags in the strings and then passing them in the UI, but idk if this would be a better solution. Let me know what you think.